SurveyApp/Models/Survey/SurveyResponse.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  SurveyResponse.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/25/21.
+//
+
+import Foundation
+
+struct SurveyAttributes: Codable {
+    
+    enum CodingKeys: String, CodingKey {
+        case title, description
+        case thankEmailAboveThreshold = "thank_email_above_threshold"
+        case thankEmailBelowThreshold = "thank_email_below_threshold"
+        case isActive = "is_active"
+        case coverImageUrl = "cover_image_url"
+        case createdAt = "created_at"
+        case activeAt = "active_at"
+        case inactiveAt = "inactive_at"
+        case surveyType = "survey_type"
```

Comment:
please take a look at https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase I think it can be implemented at NetworkAPI class. I think Toyota Wallet project has this example.

Reply:
@blyscuit I updated the code. Please help review again. Thank you in advance 🙏